packages/utilities/README.md

```diff
@@ -175,17 +175,66 @@ class RateLimitExample {
 ## NotBot
 
 ```ts
-...
 @SimpleCommand("hello")
 @Guard(NotBot)
 hello({ message }: SimpleCommandMessage): void {
   message.reply("It worked!");
 }
-...
 ```
 
 This will work on both Slash and Simple commands
 
+## PermissionGuard
```

Comment:
add this in index at top

Reply:
can you explain?

Reply:
never mind, i figured out what you meant